spec/server/unit/user.spec.coffee

```diff
@@ -56,15 +56,43 @@ describe 'User', ->
     done()
 
   describe '.updateServiceSettings()', ->
-    makeMC = (callback) ->
-
-    it 'uses emails to determine what to send to MailChimp', (done) ->
-      spyOn(mc.lists, 'subscribe').and.callFake (params) ->
-        expect(JSON.stringify(params.merge_vars.groupings[0].groups)).toBe(JSON.stringify(['Announcements']))
-        done()
+    it 'uses emails to determine what to send to MailChimp', utils.wrap (done) ->
+      email = 'tester@gmail.com'
+      user = new User({emailSubscriptions: ['announcement'], @email, emailLower: email})
+      spyOn(user, 'updateMailChimp').and.returnValue(Promise.resolve())
+      yield user.updateServiceSettings()
+      expect(user.updateMailChimp).toHaveBeenCalled()
+      done()
 
-      user = new User({emailSubscriptions: ['announcement'], email: 'tester@gmail.com'})
-      User.updateServiceSettings(user)
+    it 'updates stripe email iff email changes on save', utils.wrap (done) ->
+      stripeApi = require('../../../server/lib/stripe_utils').api
+      spyOn(stripeApi.customers, 'update')
+      user = new User({email: 'first@email.com'})
+      yield user.save()
+      
+      user = yield User.findById(user.id)
+      user.set({email: 'second@email.com'})
+      yield user.save()
+      
+      user = yield User.findById(user.id)
+      user.set({stripe: {customerID: '1234'}})
```

Comment:
I'd include an email here, just identical to the one used before. I _think_ you can update other user properties and the `save()` will include the unchanged email.